packages/core/forms/src/components/fields/FieldAutoSuggestV2.vue

```diff
@@ -42,7 +42,7 @@
 
       <template #selected-item="{ item }">
         <span class="selected-entity-item">
-          <span class="selected-entity-label">{{ item.label || '–' }}</span>
+          <span class="selected-entity-label">{{ item.label === LABEL_PLACEHOLDER ? item.value : item.label }}</span>
```

Comment:
Is `"-"` a valid name? If it is, I think we should display it as-is since the user explicitly chose that name.

Reply:
`-` is a valid one, but if we do so, what should be shown for those with name set to `undefined` 🤔

Reply:
Made a mistake, `-(short dash)` is different than `–(long dash)` , and `–(long dash)` is prohibited to be used as the entities' name. So I think the current implementation is ok for distinguishing entities' without name and user given `-(short dash)`